unittests/finality_tests.cpp

```diff
@@ -746,15 +746,15 @@ BOOST_FIXTURE_TEST_CASE(finality_skip, finality_test_cluster<4>) { try {
    // `generation == 4`. The other policies must have been overwritten since they all
    // became `pending` at the same block.
    //
-   // we need another 3-chain to make that block final.
+   // we need another 2-chain to make that block final.
    // -------------------------------------------------------------------------------
-   for (size_t i=0; i<3; ++i) {
+   for (size_t i=0; i<(num_chains_to_final - 1); ++i) {
```

Comment:
If we need a 2-chain, why `num_chains_to_final - 1`.  Also the comment at line 733 above is incorrect, we won't have three pending policies at the same time.

Reply:
Fixed it by https://github.com/AntelopeIO/spring/pull/389/commits/c1feb767a7ede694db9dae81d0af3ccf57721936

Thanks.